dev/install-common-deps.sh

```diff
@@ -28,7 +28,8 @@ conda create -q -n test-environment python=3.6
 source activate test-environment
 
 python --version
-pip install --upgrade pip==19.3.1
+pip install --upgrade pip
+pip --version
```

Comment:
Note that this change won't affect the cross version tests because we don't use `dev/install-common-deps.sh` in `cross-version-tests.yml`